packages/clay-form/src/SelectBox.tsx

```diff
@@ -164,9 +156,9 @@ const ClaySelectBox: React.FunctionComponent<IProps> = ({
 	);
 
 	return (
-		<div {...otherProps} className={classNames(className, 'form-group')}>
```

Comment:
Since this is called a `Select` component, I would expect the ...otherProps to apply to the select element and not the container. If we need the ability to add props to the container, we can use `containerProps`. This is really only for the case when the component name is not consistent with the container element.

Reply:
Alright, good to know! Makes sense now that you put it like that but I kinda forgot about `containerProps`